qcodes/dataset/experiment_settings.py

```diff
@@ -1,22 +1,23 @@
 """Settings that are indirectly related to experiments."""
 
 from typing import Optional
-from qcodes.dataset.sqlite.connection import ConnectionPlus, atomic_transaction
+from qcodes.dataset.sqlite.connection import ConnectionPlus
+from qcodes.dataset.sqlite.queries import get_last_experiment
 
 # The default experiment's exp_id. This changes to the exp_id of a created/
 # loaded experiment.
-default_experiment: Optional[int] = None
+_default_experiment: Optional[int] = None
 
 
 def _set_default_experiment_id(exp_id: int) -> None:
     """
-    Sets the default_experiment to the exp_id of a created/ loaded experiment.
+    Sets the _default_experiment to the exp_id of a created/ loaded experiment.
```

Comment:
There is no need to change the docstring to refer to the private variable. The private variable is exactly private since no one on the outside should know about it